pkgs/applications/misc/feedbackd/default.nix

```diff
@@ -59,13 +62,16 @@ stdenv.mkDerivation rec {
     pkg-config
     vala
     wrapGAppsHook3
+    cmake
+    umockdev
```

Comment:
Is `cmake` a new dependency due to "Switch all git submodules to meson subprojects" ([release notes](https://phosh.mobi/releases/rel-0.41.0/#phosh-0410-1))?

If they have stopped using git submodules then we could probably remove `fetchSubmodules = true;` from the source fetchers.


And is `umockdev` required for building or unit tests? If the latter then it should be in `nativeCheckInputs`.

Reply:
- No idea why but it builds without `cmake` now so I removed it from all the packages
- Moved `umockdev` to `nativeCheckInputs` since it's used for tests
- Removed unnecessary `fetchSubmodules`

Reply:
Thanks